crates/extrinsics/src/instantiate.rs

```diff
@@ -165,7 +166,7 @@ impl InstantiateCommandBuilder<state::ExtrinsicOptions> {
         let transcoder = artifacts.contract_transcoder()?;
         let data = transcoder.encode(&self.opts.constructor, &self.opts.args)?;
         let signer = self.opts.extrinsic_opts.signer()?;
-        let url = self.opts.extrinsic_opts.url_to_string();
+        let url = url_to_string(self.opts.extrinsic_opts.url());
```

Comment:
All usages of `url()` are wrapped in `url_to_string` so you could just make `url()` return the formatted `String` directly?

![image](https://github.com/paritytech/cargo-contract/assets/75586/850cd1da-99e4-4579-a75a-115e1cf67281)

Reply:
Done